fecgov/fec-cms
# Feature/2693 update 404 page

## Summary

- Resolves #2693 
Updating the language (and links) for the 404 page.

## Impacted areas of the application
Only fec/templates/404.html changed (though it looks like package-lock did, too, for some reason)

## Screenshots
The page:
![image](https://user-images.githubusercontent.com/26720877/54142326-d290c280-43fd-11e9-8c9d-c372ab8845fa.png)
With the feedback form open:
![image](https://user-images.githubusercontent.com/26720877/54142361-e2a8a200-43fd-11e9-8165-6d72037bb1d0.png)

## Related PRs
(none)

## How to test
- Pull the branch
- Change `DEBUG=False` (fec/fec/settings/dev.py)
- Add `ALLOWED_HOSTS = ['*']` to ⬆️
- Run `python manage.py collectstatic --noinput -v 3`
- Run the server like normal (`./manage.py runserver`)
- Check localhost:8000/some-gibberish
- Click the "feedback box" link to make sure the feedback form opens
- Click it again to toggle it closed
- Verify the "contact the FEC" link
____
